CHANGES.md

```diff
@@ -1,12 +1,17 @@
 wavesurfer.js changelog
 =======================
 
-6.5.0 (unreleased)
+6.5.0 (11.03.2023)
 ------------------
 - Regions plugin:
   - Improved delta calculation (resize end) (#2641)
 - Fix iphone silent switch webaudio mute (#2667)
 - Respect mute state when changing volume during mute (#2502)
+- Proxy HTMLMediaElement's 'waiting' event through MediaElement backend. (#2691)
+- Chores:
+  - Docs: update the issue template and wavesurfer links (#2671)
+  - Add FUNDING.yml to enable GitHub Sponsors (#2668)
+  - update dev dependencies
 
```

Comment:
no need to add the update dev dependencies line

Reply:
Why? It's good to know that the tooling is updated if you're a contributor.

Reply:
you can assume it's always updated (usually several times during dev cycle of a specific version), 100% of the time, that's why I would exclude it. If you don't want to exclude it, please start sentence with capital and use full sentence, e.g. `Update development dependencies`

Reply:
I've changed it as requested, and appreciate the attention to detail, but I think we should work out a less pedantic/more pragmatic approach to pull request reviews in the future.

The reviewer is there to help and support the contributor with expert knowledge of the codebase and an encouraging word. If we make it harder to contribute by requesting several rounds of tiny changes, pull requests will be more likely to sit stale for months.